src/model/MultiJump.java

```diff
@@ -10,16 +11,43 @@
     private int startingPosition;
     private ArrayList<SingleJump> subJumps;
 
-    public MultiJump(int startingPosition, int endingPosition, ArrayList<SingleJump> subJumps,
+    public MultiJump(int startingPosition, int endingPosition, List<Integer> intermediatePositions,
```

Comment:
It was actually for the ease of building MultiJumps, since for each SingleJump, you'd have to get the correct board to pass to it, otherwise things get wonky. Just giving it the intermediate positions it lands on is enough to build the appropriate multijump automatically.